udata/core/spatial/models.py

```diff
@@ -53,6 +58,24 @@ def __unicode__(self):
     def __html__(self):
         return gettext(self.name) + ' <i>(' + self.code + ')</i>'
 
+    def logo_url(self, external=False):
+        path = ((self.logo.filename
+                 and self.logo.fs.exists(self.logo.filename)
+                 and self.logo())
+                or '')
+        if path:
+            # Any attempt to put the `external=True` parameter within
```

Comment:
My bad, I need to publish the repository.
You can have an external URL with `self.logo.best_url(external=True)` or with `self.logo.fs.url(self.logo.filename, external=True)`